app/posts/register-trainee-teachers/2020-09-07-register-teacher-trainees-alpha-prototype.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Register Trainee Teachers - alpha prototype 
```

Comment:
```suggestion
title: Initial alpha static prototype 
```

Register Train Teachers isn’t necessary as already in that section. Maybe distinguish this prototype from the functional one you are developing now.